Add note to docs clarifying storage buckets have to be created manually (#8160)
grafana/mimir: docs/sources/mimir/configure/configure-object-storage-backend.md

# Configure Grafana Mimir object storage backend

> Grafana Mimir can use different object storage services to persist blocks containing the metrics data, as well as recording rules and alertmanager state. If one of these services is used, the configured storage bucket will have to be created beforehand, as Grafana Mimir will not create it on its own.

Reviewer: While we are here, we can fix the spelling of Alertmanager.

I think the key information you are trying to convey is that Mimir doesn't create the object storage buckets.

```suggestion
Grafana Mimir can use different object storage services to persist blocks containing the metrics data, as well as recording rules and Alertmanager state.

Mimir doesn't create the configured storage bucket, you must create it yourself.
```
